cipher-ctr-mt-functions.c

```diff
@@ -281,9 +280,15 @@ thread_loop(void *job)
 	 * a draining queue to become empty.
 	 *
 	 * Multiple threads may be waiting on a draining queue and awoken
-	 * when empty.  The first thread to wake will mark it as filling,
+	 * when empty. qThe first thread to wake will mark it as filling,
```

Comment:
Typo introduced here in commit [2c08ebf](https://github.com/rapier1/hpn-ssh/pull/61/commits/2c08ebf449be298311d180a98c0e5bb1df3150a4).